.github/scripts/checkKnipRegression.ts

```diff
@@ -25,31 +29,43 @@ function compareKnipResults(
   const rows: Array<Record<string, string | number>> = []
   for (const [category, baseProblems] of Object.entries(baseKnipResults)) {
     const branchProblems = branchKnipResults[category] ?? 0
+    const hasRegression = branchProblems > baseProblems
     rows.push({
-      'Issue Category': category.trim(),
-      'Base Branch': baseProblems,
-      'PR Branch': branchProblems,
+      'Issue Category': hasRegression ? chalk.red(category.trim()) : category.trim(),
+      'Base Branch': hasRegression ? chalk.green(baseProblems) : baseProblems,
+      'PR Branch': hasRegression ? chalk.red(branchProblems) : branchProblems,
     })
-    if (branchProblems > baseProblems) {
+    if (hasRegression) {
       shouldFail = true
     }
   }
-  console.table(rows)
+
+  // Use easy-table instead of console.table, because console.table doesn't support colors (it escapes them)
+  console.log(easyTable.print(rows))
   return shouldFail
 }
 
 if (process.env.GITHUB_EVENT_NAME === 'pull_request') {
-  const branchKnipOutput = $.exec('yarn knip --no-gitignore').stdout.trim()
+  const branchKnipOutput = stripAnsi($.exec('yarn knip --no-gitignore').stdout.trim())
```

Comment:
Wonder if we should silence the entire knip output (with a silent: true option) and only print the diff. I've found this noisy and there's a ton of duplicates with both the PR branch and base outputs.

Reply:
Yes it is a noisy, but maybe we can do that separately if end up using the json reporter?
Now that we have a summary of the issues having the full output further above can maybe help debug/ensure things are working as expected?